### PR TITLE
Adding TMI

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Added TMI as a dependency to Toffy. Removed Ark as a dependency.

**How did you implement your changes**

Substituted utility imports from ark with tmi.

**Remaining issues**

- [x] Merge the branch on TMI angelolab/tmi#13

### CODE REVIEW

File: toffy/qc_comp.py [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
-import ark.utils.io_utils as io_utils
-import ark.utils.misc_utils as misc_utils
-import ark.utils.load_utils as load_utils
-from mibi_bin_tools import bin_files
+from toffy.mibitracker_utils import MibiRequests, MibiTrackerError
 
-
-# needed to prevent UserWarning: low contrast image barf when saving images
-import warnings
 warnings.filterwarnings('ignore', category=UserWarning)
```
Comment (ngreenwald) [inline toffy/qc_comp.py:19]: We don't need this anymore because of `save_image` right?

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
-ark-analysis @ git+https://github.com/angelolab/ark-analysis.git@v0.4.4
-mibi-bin-tools @ git+https://github.com/angelolab/mibi-bin-tools.git@v0.2.4
-tmi @ git+https://github.com/angelolab/tmi.git
+mibi-bin-tools @ git+https://github.com/angelolab/mibi-bin-tools.git@v0.2.5
+tmi @ git+https://github.com/angelolab/tmi.git@v0.0.1
```
Comment (ngreenwald) [inline requirements.txt:2]: The conda file as well, right?

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looking good

File: toffy/qc_comp.py (hunk 1 above)
Comment (srivarra) [reply toffy/qc_comp.py:19]: Yeah, I'll remove it.

File: requirements.txt (hunk 2 above)
Comment (srivarra) [reply requirements.txt:2]: Conda file needs the updated requirements as well.

**Review by ngreenwald — APPROVED** [review]
Nice!

**Pull request merged**